packages/ra-ui-materialui/src/button/SaveButton.spec.tsx

```diff
@@ -45,50 +41,8 @@ describe('<SaveButton />', () => {
                 />
             </TestContext>
         );
-        fireEvent.click(getByLabelText('ra.action.save'));
-        expect(onSubmit).toHaveBeenCalled();
-    });
-
-    it('should trigger submit action when Enter key if no saving is in progress', () => {
```

Comment:
Why did you remove this test?

Reply:
It never worked, And I never understood why. I replaced it by an e2e doing same